server.js

```diff
@@ -86,21 +97,30 @@ var createAndLogEvent = function(data, req) {
     var value = parseValue(properties[key]);
     entry += ' ' + key + '=' + value;
   }
-  entry += '\n';
-  fs.appendFile(path.resolve(__dirname, './events.log'), entry, function(err) {
-    if (err) {
-      console.log(err);
-    } else {
-      //console.log("Logged tracked data");
-    }
-  });
+  if (argv.docker) {
+    console.info(entry);
+  } else {
+    entry += '\n';
+    fs.appendFile(path.resolve(__dirname, './events.log'), entry, function(err) {
+      if (err) {
+        console.error(err);
+      } else if (argv.debug) {
+        console.log('Logged tracked data');
+      }
+    });
+  }
 };
 
 /*
  * Use Middlewares
  **********************************
  */
-app.use(express.logger());
+if (argv.docker) {
+  var morgan = require('morgan');
+  app.use(morgan('combined', { stream: process.stderr }));
```

Comment:
@elementalvoid Yes, exactly. That's what this does. This line is for the server logs. I just wasn't sure if we wanted to capture them. It creates the Apache combined log and sends it to stderr. Up above in the createAndLogEvent method the event is sent to console.info which goes to stdout.